tests/test_array_namespace.py

```diff
@@ -19,15 +19,10 @@ def test_array_namespace(library, api_version):
     else:
         assert namespace == getattr(array_api_compat, library)
 
-def test_array_namespace_multiple():
-    import numpy as np
-
-    x = np.asarray([1, 2])
-    assert array_namespace(x, x) == array_namespace((x, x)) == \
-        array_namespace((x, x), x) == array_api_compat.numpy
 
 def test_array_namespace_errors():
     pytest.raises(TypeError, lambda: array_namespace([1]))
+    pytest.raises(TypeError, lambda: array_namespace([1, 2]))
```

Comment:
Isn't this the same as the test above? A better test would be that the lines you deleted above raise an error.

Reply:
Done 👍